src/pages/EMPReplicantTargets.page

```diff
@@ -0,0 +1,18 @@
+<apex:page standardController="EMP_Organization_Member__c" extensions="EMPReplicantProfile" sidebar="false" language="es">
+<apex:form >
+    <apex:pageBlock >
+      <apex:pageBlockSection title="Metas">
+        <apex:pageBlockTable value="{!targetDiagnostic}" var="fTD">
+            <apex:column headerValue="Fecha Visita">
+            <apex:outputLink target="_blank" value="/{!fTD.Id}">{!fTD.Visit_Date__c}</apex:outputLink>
```

Comment:
I'm not sure what you mean - Do you mean showing the date in a different date format?

If so, you can do something like this:
```xml
<apex:outputText value="{0, date, MMMM d','  yyyy}">
    <apex:param value="{!fTD.Visit_Date__c}" />
</apex:outputText>
```
Should also work with output link.
Information here: http://www.salesforce.com/us/developer/docs/pages/Content/pages_compref_outputText.htm